sdks/python/container/Dockerfile

```diff
@@ -38,11 +36,10 @@ RUN apt-get update && \
 # Install required packages for Beam Python SDK and common dependencies used by users.
 ####
 
-# SDK dependencies not listed in base_image_requirements.txt will be installed
-# when we install SDK with pip below.
 COPY target/base_image_requirements.txt /tmp/base_image_requirements.txt
 RUN \
-    pip install -r /tmp/base_image_requirements.txt && \
+    # use --no-deps to ensure the list includes all transitive dependencies.
+    pip install --no-deps -r /tmp/base_image_requirements.txt && \
```

Comment:
Could it be the case that future deps will cause future issues?  I mean if future deps come in will that be a problem?

Reply:
it's always a possibility that future deps will cause some problems; but as far as this line of code is  concerned, future deps should also be included in base_image_requirements, after the requirement list is regenerated.